libbeat/_meta/config.full.yml

```diff
@@ -112,6 +112,10 @@ output.elasticsearch:
   # Optional HTTP Path
   #path: "/elasticsearch"
 
+  # Custom HTTP headers to add to each request
+  #headers:
+  #  X-My-Header: Contents of the header
```

Comment:
Maybe mention somewhere in the config file that you can pass multiple values separated by comma.